datamade/la-metro-councilmatic
# fix `Significant Date` facet and add `Fiscal Year` facet

## Overview

See title

### Checklist



## Testing Instructions

[for @fatima3558 to do]
 * Deploy to staging by shelling into the lametro server
 * run the `refresh_guid` command on the staging server
 * visit the staging site and check that the `Significant Date` facet is properly populated

Handles #716 
Handles #370
